tests/unit/s2n_handshake_test.c

```diff
@@ -119,13 +119,13 @@ int test_cipher_preferences(struct s2n_config *server_config, struct s2n_config
         }
 
         /* Craft a cipher preference with a cipher_idx cipher
-           NOTE: Its safe to use memcpy as the address of server_cipher_preferences
+           NOTE: Its safe to use EXPECT_MEMCPY_SUCCESS as the address of server_cipher_preferences
```

Comment:
These comments are an argument for NOT using the macro :) You should just remove them.